src/background_jobs.rs

```diff
@@ -28,8 +28,6 @@ pub struct Environment {
     http_client: AssertUnwindSafe<Client>,
 }
 
-// FIXME: AssertUnwindSafe should be `Clone`, this can be replaced with
-// `#[derive(Clone)]` if that is fixed in the standard lib
```

Comment:
Implementing `Clone` doesn't seem straightforward: https://github.com/rust-lang/rust/pull/81258
I think we could remove this for now.